scripts/dag_fact.coffee

```diff
@@ -9,7 +9,8 @@
 
 facts = [
   ":belle-dag: Belle loves sunbeams more than anything except belly rubs and bacon.",
-  ":belle-dag: Belle will not eat cheese. There may be pills hiding in there.",
```

Comment:
Is this one supposed to be deleted?

Reply:
Yes. Belle has made her peace with cheese.